### DISCUSSION TITLE
AUR support?

### DISCUSSION THREAD

**Discussion by Skukkuk** [BepInEx/BepInEx#359] [Help and Q&A] [answered]
Hi, 

just wanted to mention Arch Linux installations work differently. Is there any kind of installation over AUR possible? 

I mean, if I install BepInEx from AUR I do get steady updates and do not have to do manually for every single update. 

Cheers!

**Comment by ghorsington** [comment, marked as answer]
Greetings!

> just wanted to mention Arch Linux installations work differently.

Would you mind elaborating? I've been using BepInEx on Arch, and various Arch-based distributions (e.g. Manjaro, EndeavourOS) and so far hasn't noticed having to do any extra installation steps in addition to those mentioned in [BepInEx installation guide](https://docs.bepinex.dev/articles/user_guide/installation/index.html?tabs=tabid-nix). Of course, some games require additional setup, but that generally does not depend on OS.

> Is there any kind of installation over AUR possible?

Please correct if I am wrong, but isn't AUR meant for system-wide packages (e.g. programs, system-wide libraries and tools)? BepInEx is not a global tool but rather a loader that you need to install separately for each game. Instead, it would make more sense to have some BepInEx installer or a mod manager on AUR. For example, one could probably ask [r2modman](https://github.com/ebkr/r2modmanPlus) maintainers to publish an install script on AUR.

**Reply by Windows10CE** [reply]
Although there's currently no version on the AUR, there is a prebuilt version of r2modman that can be installed with `pacman -U` on the release page, or an AppImage if you prefer that

**Comment by Skukkuk** [comment]
@ghorsington 
Thank you for the fast answer. Well, I didn't consider global and game-wise installation of BepInEx. My bad. Still wondering if I have to manually update BepInEx or if it just provides libraries for once. So I will try the manual installation for the Steam build of Humankind next and crossing fingers that I won't have to do the same process every other week. 

@Windows10CE 
Ok, I think I got the answer from ghosington, so global installation might not work for my purpose on a single game. Thx anyway for pointing me into the AUR options.
